versions.csv

```diff
@@ -1,6 +1,6 @@
 AssemblyPath, Version
-iothub\device\src\Microsoft.Azure.Devices.Client.csproj, 1.33.0
-iothub\service\src\Microsoft.Azure.Devices.csproj, 1.28.0
+iothub\device\src\Microsoft.Azure.Devices.Client.csproj, 1.33.1
+iothub\service\src\Microsoft.Azure.Devices.csproj, 1.28.1
```

Comment:
Do we bump version for logging?

Reply:
Oh I see you meant logging support.

Reply:
Yeah, the newer version of the sdk will have these log statements in them => we need to release it.